### ISSUE TITLE
Add tests and badges

### ISSUE THREAD

**Issue by ddehueck** [ddehueck/jax-loss#1]
Add tests for each loss function - comparing to pytorch and tensorflow implementations.

Could be nice to have badges as described in: https://thomas-cokelaer.info/blog/2014/08/1013/

**Comment by ddehueck** [comment]
Resources for Travis-CI https://docs.travis-ci.com/user/tutorial/
